gtk/conf.h

```diff
@@ -23,24 +23,27 @@
 #pragma once
 
 #include <inttypes.h>
+#include <string>
+#include <vector>
+
 #include <libtransmission/transmission.h> /* tr_variant, tr_session */
 #include <libtransmission/quark.h>
 
-void gtr_pref_init(char const* config_dir);
+void gtr_pref_init(std::string const& config_dir);
```

Comment:
_(opinion)_ std::string_view might be a better type here

Reply:
Called in one place where `std::string` is already available, and must be copied inside, not sure what the advantage would be.